pkgs/tools/inputmethods/interception-tools/default.nix

```diff
@@ -1,32 +1,29 @@
-{ lib, stdenv, fetchurl, pkg-config, cmake, libyamlcpp,
-  libevdev, udev }:
+{ lib, stdenv, fetchFromGitlab, pkg-config, cmake, libyamlcpp,
+  libevdev, udev, boost }:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitLab, pkg-config, cmake, libyamlcpp, libevdev, udev, boost }:
```

Reply:
I used the wrong command to test in my local machine, so I ended up testing the older version instead of catching this mistake. Please accept my sincere apology.